clover.rb

```diff
@@ -2,11 +2,6 @@
 
 require_relative "model"
 
-unless defined?(Unreloader)
```

Comment:
Normally we define `Unreloader` in `config.ru`. But rspec uses Clover directly without `config.ru`. We need this redefining only for tests, so I moved it to spec_helper